templates/install_puppet.bat.erb

```diff
@@ -14,8 +14,7 @@ if exist %pid_path% del %pid_path%
 
 :wait_for_pid
 timeout /t 5 /nobreak > NUL
-FOR /F "tokens=*" %%A IN ('tasklist /FI "PID eq %AGENT_PID%" /NH') DO set _task=%%A
-echo %_task% | findstr "No tasks are running" >nul
+wmic path Win32_Process where handle=%AGENT_PID% get handle /format:textvaluelist | findstr /I "Handle=%AGENT_PID%"
```

Comment:
We tested that `"Handle=PID"` appears on English and German... not sure about French or other languages.

Since it's a WMI property, I doubt that it's localized - but we might make the code a little more resilient if we just look for `=PID` instead of `Handle=PID`.

Thoughts @puppetlabs/windows